### ISSUE TITLE
Report not updated during obs-status disposition process

### ISSUE THREAD

**Issue by javierggt** [sot/agasc#139]
During [obs-status disposition](https://sot.github.io/agasc/supplement.html#applying-review), the weekly report is expected to be updated reflecting the fact that a star-observation is marked as bad in the updated supplement (changing the row color from red to yellow). This did not happen when dispositioning the failed star-obsevation on [2022:136](https://cxc.cfa.harvard.edu/mta/ASPECT/agasc/supplement_reports/weekly/2022:136), although the tables in the supplement seem to have been updated correctly, and the observation is skipped after disposition as expected.

In other words: the star-obs was properly excluded, there were no other observations to process, and the report was not updated (perhaps because there were no other observations).
